libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/WifiScanner.java

```diff
@@ -153,13 +156,11 @@ synchronized void activatePeriodicScan(final ActiveOrPassiveStumbling stumblingM
         // Ensure that we are constantly scanning for new access points.
         mWifiScanTimer = new Timer();
         mWifiScanTimer.schedule(new TimerTask() {
-            int mPassiveScanCount;
 
             @Override
             public void run() {
                 if (stumblingMode == ActiveOrPassiveStumbling.PASSIVE_STUMBLING &&
-                        mPassiveScanCount++ > AppGlobals.PASSIVE_MODE_MAX_SCANS_PER_GPS) {
```

Comment:
Sorry, I do not understand this comment either.
In a1cbaf5392fb824c9bb8f6ff4d7de3c08c5d8752 I made the counter a member of the scanner so that the variable can be reset in `WifiScanner.start()` to keep the TimerTask alive. That way it will scan data for more than one location if there is a new one already.
This is not possible if the variable is a member of the TimerTask (as far as I can see). Otherwise it would be the same as before, or not?